lib/modules/platform/azure/index.ts

```diff
@@ -990,3 +994,42 @@ export async function deleteLabel(
   const azureApiGit = await azureApi.gitApi();
   await azureApiGit.deletePullRequestLabels(config.repoId, prNumber, label);
 }
+
+function cachePr(pr?: GitPullRequest): void {
+  config.prList ??= [];
+  // istanbul ignore if
```

Comment:
```suggestion
```

Don't ignore code which is easy to test. But why not make this param mandatory?

Reply:
copy paste from github. no idea, ill try